website_cms/templates/menu.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<!-- User Navbar -->
+
+<template id="add_cms_user_actions" inherit_id="website.user_navbar">
+
+  <!-- edit / translate buttons have weird  conditions: protect them better.
+  For instance, the homepage controller overrides `editable` and `translatable`
+  to allow only `bwi_website.group_homepage_manager` group to update it. -->
+  <xpath expr="//ul[@id='oe_editzone']" position="after">
+    <t t-call="website_cms.cms_user_actions" />
+  </xpath>
+
+</template>
+
+
+<template id="cms_user_actions">
+
+  <ul class="cms-main-menu ml8 nav navbar-nav hidden-xs">
+    <t t-if="website.is_cms_page(main_object)">
+      <t t-if="website.cms_can_edit(main_object=main_object)">
+        <li class="edit edit-frontend ml16">
+          <a title="Edit metadata (name, description, etc)" class="btn btn-link btn-xs"
+             t-att-href="website.cms_edit_link(main_object=main_object)">
+            <i class="fa fa-pencil-square-o" /> Metadata</a>
```

Comment:
if you have `t-` attributes the element is ignored, see [here](https://github.com/OCA/OCB/blob/9.0/openerp/tools/translate.py#L256). I think I can use `t-ignore`.